.github/workflows/build_docker_image.yml

```diff
@@ -33,32 +33,35 @@ jobs:
     steps:
 
     - name: Login to GitHub Container Registry
-      uses: docker/login-action@v2
+      uses: docker/login-action@v3
       with:
         registry: ghcr.io
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
     - name: Build and push
-      uses: docker/build-push-action@v3
+      uses: docker/build-push-action@v6
       with:
         push: ${{ github.ref == 'refs/heads/main' }}
         tags: ghcr.io/f3d-app/f3d-wasm:latest
         file: webassembly/Dockerfile
+        build-args: |
+          CMAKE_BUILD_PARALLEL_LEVEL=4
```

Comment:
specific reason to bring it down to 4 ?

Reply:
Linux runners on github have 4 CPUs now.

Reply:
ha nice, maybe we should change that in our CI to improve build time